README.rst

```diff
@@ -76,7 +76,7 @@ Installation
 
      .. code:: sh
 
-        docker pull neuropoly/sct:sct-4.0.0-official
+        docker pull neuropoly/sct:sct-latest
```

Comment:
i would recommend to use `latest` instead of `sct-latest` (for simplicity)
same comment for all other versions.